pnpm-workspace.yaml

```diff
@@ -1,3 +1,4 @@
 packages:
   - 'packages/*'
   - 'website'
+  - 'demos/*'
```

Comment:
Demos should be outside the root project because it requires a different package manager (npm) because of expo.
```suggestion
```